README.md

```diff
@@ -1,2 +1,21 @@
-# js-weather-app
-Microverse Weather App with JavaScript
+# Project: Weather App with JavaScript
+
+This project consists in creating a Weather APP using the API of OpenWeather. You can find more about the requirements [here](https://www.theodinproject.com/courses/javascript/lessons/weather-app).
+
+## Built With
+
+- JavaScript
+- Bootstrap
+- Webpack
+
```

Comment:
You can add the steps to `git clone` your project to make the README file more detailed.